package.json

```diff
@@ -59,6 +58,10 @@
     "@babel/preset-env": "7.25.3",
     "@babel/preset-react": "7.24.7",
     "@babel/preset-typescript": "7.24.7",
+    "@biomejs/biome": "1.9.2",
+    "@canonical/biome-config": "0.1.1-experimental.0",
+    "@canonical/ds-react-core": "0.1.1-experimental.0",
```

Comment:
This package will eventually have the new React components themselves, but it doesn't contain much that is ready for consumption right now.